lightning/src/chain/channelmonitor.rs

```diff
@@ -411,7 +411,8 @@ enum OnchainEvent {
 		/// If the funding spend transaction was a known remote commitment transaction, we track
 		/// the output index and amount of the counterparty's `to_self` output here.
 		///
-		/// This allows us to generate a [`Balance::CounterpartyRevokedOutputClaimable`] for the
+		/// This allows us to generate a [`Balance::CounterpartyRevokedOutputClaimable`] for
+        /// thechaannmo
```

Comment:
typo: `/// thechaannmo`, also should use [spaces](https://github.com/lightningdevkit/rust-lightning/blob/main/CONTRIBUTING.md#coding-conventions) not tabs (though the full line will be deleted so it doesn't matter much here).